python/delete-default-vpcs.py

```diff
@@ -19,12 +20,18 @@ def json_serial(obj):
     raise TypeError("Type not serializable")
 
 class iam:
-    def __init__(self, profile, dry_run):
+    def __init__(self, profile, mgmt_account_number, account_number, stage, deployment_type, dry_run):
```

Comment:
Is `mgmt_account_number` used anywhere? Sorry if I'm just missing something!

Reply:
thanks - now removed